components/email-mgt/org.wso2.carbon.email.mgt/src/main/java/org/wso2/carbon/email/mgt/store/dao/OrgNotificationTemplateDAO.java

```diff
@@ -60,12 +67,20 @@ public void addNotificationTemplate(NotificationTemplate notificationTemplate, i
 
         NamedJdbcTemplate namedJdbcTemplate = JdbcUtils.getNewNamedJdbcTemplate();
         try {
-            namedJdbcTemplate.executeInsert(INSERT_ORG_NOTIFICATION_TEMPLATE_SQL, (preparedStatement -> {
+            String insertOrgNotificationTemplateSql = isUnicodeDataType ? INSERT_ORG_NOTIFICATION_TEMPLATE_SQL_UNICODE :
+                    INSERT_ORG_NOTIFICATION_TEMPLATE_SQL;
+            namedJdbcTemplate.executeInsert(insertOrgNotificationTemplateSql, (preparedStatement -> {
                 preparedStatement.setString(TEMPLATE_KEY, locale.toLowerCase());
                 preparedStatement.setString(LOCALE, locale);
-                preparedStatement.setString(SUBJECT, notificationTemplate.getSubject());
-                preparedStatement.setString(BODY, notificationTemplate.getBody());
-                preparedStatement.setString(FOOTER, notificationTemplate.getFooter());
+                if (isUnicodeDataType) {
+                    preparedStatement.setNString(SUBJECT, notificationTemplate.getSubject());
```

Comment:
This needs to be changes as
```suggestion
                    preparedStatement.setNString(NSUBJECT, notificationTemplate.getSubject());
```

right?